CAP/gap/ConstructiveCategoriesRecord.gi

```diff
@@ -18,7 +18,9 @@ IsEnrichedOverCommutativeRegularSemigroup := Concatenation(
 IsAbCategory := Concatenation( [ 
   "ZeroMorphism", 
   "IsZeroForMorphisms", 
-  "AdditiveInverseForMorphisms" ], ~.IsEnrichedOverCommutativeRegularSemigroup ),
+  "SubtractionForMorphisms",
+  "AdditiveInverseForMorphisms",
+  "MultiplyWithElementOfCommutativeRingForMorphisms" ], ~.IsEnrichedOverCommutativeRegularSemigroup ),
```

Comment:
Yes, sorry, I misinterpreted the piece of code I see above. This list `IsEnrichedOverCommutativeRegularSemigroup` is appended at the end.